app/cafe.py

```diff
@@ -0,0 +1,27 @@
+import datetime
+
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> None | str:

```

Comment:
There is no case in the function that will make it return None. You can change return annotation to `-> str`